assets/monitoring/grafana/v1alpha1/grafanadashboards.go

```diff
@@ -63,6 +72,13 @@ func NewGrafanaDashboardsFromFS(filesystem embed.FS, root string) (GrafanaDashbo
 			continue
 		}
 
+		// Exclude platform dashboards that are known to cause issues.
+		// TODO: get rid of this exclusion once the root cause is fixed (https://github.com/scylladb/scylla-operator/issues/2822).
```

Comment:
less indirect language

```suggestion
		// Exclude platform dashboards that are known to cause https://github.com/scylladb/scylla-operator/issues/2822.
		// TODO: get rid of this exclusion once the root cause is fixed.
```